src/explore/AB_testing/by_date_camps.py

```diff
@@ -17,63 +17,80 @@
 4. Bandit (the models) probabilities are updated as they are played 
 '''
  
-
-from cap_ab_testing import experiment_numerical
-from cap_ab_testing_rand import experiment_numericalR
+from enum import Enum
+import logging
 import pandas as pd 
 import numpy as np 
 import matplotlib.pyplot as plt  
+import pickle, csv 
+
+from cap_ab_testing import experiment_numerical
+from cap_ab_testing_rand import experiment_numericalR
 from preprocessing_formatting import drop_cols , one_hot_encoding , scale
 from postHOCAB import run_tests 
-import pickle, csv 
+
 pd.set_option('display.max_columns', None) 
-dataframe = pd.read_csv('/home/allen/RIP_Tensor1/capstone2/Capstone2/src/explore/ab_df.csv') 
-#ids = pd.read_csv('/home/allen/RIP_Tensor1/capstone2/Capstone2/src/explore/testing/x1.csv')
-df_withdates = pd.read_csv('/home/allen/RIP_Tensor1/capstone2/Capstone2/data/placeholder/df_withdates.csv') 
-df_blank = pd.read_csv('/home/allen/RIP_Tensor1/capstone2/Capstone2/data/ready12_24_train.csv') 
 
-def edit_df():
+DATAFRAME_AB = '/home/allen/RIP_Tensor1/capstone2/Capstone2/src/explore/ab_df.csv'
+DATAFRAME_WITHDATES = '/home/allen/RIP_Tensor1/capstone2/Capstone2/data/placeholder/df_withdates.csv'
+DATAFRAME_BLANK = '/home/allen/RIP_Tensor1/capstone2/Capstone2/data/ready12_24_train.csv'
+
+_LOGGER = logging
+
+
+class CampColumns(Enum):
+    START_DATE2 = 'Camp_Start_Date2'
+    END_DATE2 = 'Camp_End_Date2'
+    CAMP_ID = 'Health_Camp_ID'
+
```

Comment:
we always use an Enum class like this to define the columns in a dataframe.  Once you get used to this style it is really nice.  Then when referencing a column you only have to remember CampColumns and the autofill will show all the column options.